### DISCUSSION TITLE
Unable to instantiate model (type=value_error)

### DISCUSSION THREAD

**Discussion by Xiaomingge666** [imartinez/privateGPT#740] [Q&A]
Using embedded DuckDB with persistence: data will be stored in: db
Found model file at  /dssg/home/acct-seety/seety/privateGPT-main/models/ggml-gpt4all-j-v1.3-groovy.bin
Invalid model file
Traceback (most recent call last):
  File "/dssg/home/acct-seety/seety/privateGPT-main/privateGPT.py", line 83, in <module>
    main()
  File "/dssg/home/acct-seety/seety/privateGPT-main/privateGPT.py", line 38, in main
    llm = GPT4All(model=model_path, n_ctx=model_n_ctx, backend='gptj', n_batch=model_n_batch, callbacks=callbacks, verbose=False)
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for GPT4All
__root__
  Unable to instantiate model (type=value_error)

**Comment by ghost** [comment]
It appears that the Bin file was designed for an earlier iteration of GPT-4-All and is not compatible with the more recent version.
Solution:
pip show gpt4all
pip uninstall gpt4all
pip show gpt4all
pip install gpt4all==0.2.3
If you come across this error with other models, consider attempting to downgrade the module you are currently utilizing.

**Reply by Chris-V-Aurea** [reply]
Thanks I had the same problem and now I get to see the prompt at least.

**Reply by ghost** [reply]
I faced additional errors and found out that running 'pip install -r requirements.txt' installed newer versions of libraries and modules. 
To resolve this, I manually installed the precise versions listed in the 'requirements.txt' file on GitHub from cmd/terminal.

**Reply by PasinBuakhaw** [reply]
But when i run privateGPT.py with gpt4all 0.2.3 version This error is appear
`GPT4All.__init__() got an unexpected keyword argument 'device' (type=type_error)` How should i do 

Thank you!!

**Comment by stccorp** [comment]
The requirement.txt says gpt4all==1.0.8, why will you go to 0.2.3??
